doc/network-api.txt

```diff
@@ -141,6 +141,11 @@ Properties	string Mode [readonly]
 			unavailable, this property will not be returned by
 			GetProperties or will be set to an empty string.
 
+		string RejectCause [readonly, optional]
+
+			Contains the current network reject cause after a failure
+			to register or activate a context on the network.
+
```

Comment:
This piece of information maybe is a signal instead of property. It is of a transient nature.

Reply:
It can be both, just like for the modem.
The modem emits an URC for each error, and the last one can be checked by querying AT^SIND="ceer",2
I believe the interface can reflect this, with the constraint that a getProperty is supposed to be "cheap", and so no query upon the module, just presentation of the latest status, as it is now.